### PR TITLE
general body scroll fix by CodingYourLife. be aware that it will make…

### PR DESCRIPTION
**Pull request by codingyourlife**
… absolutely every page scrollable.

### CODE REVIEW

File: src/data/css/common.css [hunk 1]
```diff
@@ -19,6 +19,10 @@ body.privacypopup-open {
 body.show-cookie-policy-info {
   top: 0 !important;
 }
+body {
```
Comment (obfuscated-loop) [inline src/data/css/common.css:22]: Be weary that some websites will use the `<html>` tag to set an `overflow`.

**Review by ghost — COMMENTED** [review]
the comment "fixes e.g..." is not necessarily useful
otherwise, it is very good!

**Comment by OhMyGuus** [comment]
Not planning on using this, however maybe it's an idea to add a broken scrollbar button to the report so a bot can automatically create a PR with common 85 with it.

**Comment by obfuscated-loop** [comment]
@OhMyGuus That would be a good feature, maybe you could add a `<select>` within a reporting window so users are able to choose from say 'General issue', 'Scrollbar issue' or 'Modal issue' with a brief description of what each one is after selection. I could make a P.R for this if you'd like.

**Comment by azgaresncf** [comment]
> @OhMyGuus That would be a good feature, maybe you could add a `<select>` within a reporting window so users are able to choose from say 'General issue', 'Scrollbar issue' or 'Modal issue' with a brief description of what each one is after selection. I could make a P.R for this if you'd like.

with github issue templates! ;)

**Comment by OhMyGuus** [comment]
> @OhMyGuus That would be a good feature, maybe you could add a `<select>` within a reporting window so users are able to choose from say 'General issue', 'Scrollbar issue' or 'Modal issue' with a brief description of what each one is after selection. I could make a P.R for this if you'd like.

Yeah +1, would really appreciate a pr with that inside👍

**Comment by obfuscated-loop** [comment]
> Yeah +1, would really appreciate a pr with that inside👍
Working on this now - just wondering, with regards to the notes, does your API allow for that to be null? I have gone for hiding the notes textarea when it isn't a 'General issue'. Let me know your thoughts about this. GIF attached. 
example

I have also implemented the `<select>` value as a new parameter in the API request called `issueType`.

**Comment by obfuscated-loop** [comment]
Actually I realise appending `issue` to each value is redundant and they could just be `general`, `scrollbar` or `modal`.

**Comment by obfuscated-loop** [comment]
@OhMyGuus Merry Christ/Xmas :christmas_tree: 

Updated GIF attached. Just need to add textarea validation for the general issue.
example

**Comment by OhMyGuus** [comment]
Merry Xmas 🎄
Looks good 👍 and the api is quite flexible will just change it to whatever it requires :)

**Comment by codingyourlife** [comment]
I'm not necessarily a fan of the "everything needs to be reported" mentality but with a certain plugin popularity and degree of automation I guess that works out. So whatever gets the job done! I will close this issue now...

**Pull request closed without merging**